api/types/container/config.go

```diff
@@ -34,29 +34,29 @@ type HealthConfig struct {
 // All fields added to this struct must be marked `omitempty` to keep getting
 // predictable hashes from the old `v1Compatibility` configuration.
 type Config struct {
-	Hostname        string                // Hostname
```

Comment:
@miaoyq as it's not related to the changes, could you remove this file from the commit ? 👼 (should be `git checkout HEAD~1 api/types/container/config.go`)